modules/monitoring-enabling-an-alertmanager-instance-for-user-defined-projects.adoc

```diff
@@ -0,0 +1,103 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-alerts-for-user-defined-projects.adoc
+
+:_content-type: PROCEDURE
+[id="monitoring-enabling-an-alertmanager-instance-for-user-defined-projects_{context}"]
+= Enabling an Alertmanager instance for user-defined projects
+
+A cluster administrator can enable a dedicated instance of Alertmanager to use with user-defined projects in a cluster.
+Doing so enables developers and other non-administrator users of user-defined projects to set up individual alerting configurations to suit their needs without the help of an administrator.
```

Comment:
It's still possible for developers to configure alert notifications themselves without the dedicated Alertmanager. The difference is that the same Alertmanager will be used for both platform and user-defined alerts which may not be a desired situation for cluster admins.

Reply:
I've rewritten the introduction to try and better capture the use case and benefits of the separate Alertmanager instance